### ISSUE TITLE
Downloading project from Windows results in folders with backslashes in their names rather than subdirectories

### ISSUE THREAD

**Issue by WasabiFan** [ev3dev/vscode-ev3dev-browser#48]
Windows, d3102742344feeb65d4deaa07b8c0eecfb43baad

Repro steps:
- Clone https://github.com/ev3dev/ev3dev-lang-python
- Open Code in the repository root
- Download the project

Result:

```
robot@ev3dev:~$ ls ev3dev-lang-python/
ev3dev/           ev3dev\_platform/ ev3dev\control/   ev3dev\sensor/
```

I assume there's a `.posix` missing somewhere in the file path manipulation logic.

```
========== ev3dev-sysinfo ==========
Image file:         ev3dev-stretch-ev3-generic-2017-09-30
Kernel version:     4.9.49-ev3dev-1.5.1-ev3
Brickman:           0.10.0
ev3devKit:          0.5.2
Board:              board0
BOARD_INFO_HW_REV=7
BOARD_INFO_MODEL=LEGO MINDSTORMS EV3
BOARD_INFO_ROM_REV=6
BOARD_INFO_SERIAL_NUM=00165340720B
BOARD_INFO_TYPE=main
```

**Comment by WasabiFan** [comment]
It's worth noting that this worked previously, at least in some form; it was only when I pulled earlier today that this began occurring.

**Comment by dlech** [comment]
Probably introduced in https://github.com/ev3dev/vscode-ev3dev-browser/commit/792800aa6cb12be40418a2ae0cc37cd00b4fc9b6#diff-4ac32a78649ca5bdd8e0ba38b7006a1e

**Comment by WasabiFan** [comment]
Interestingly, I just tested ac44e5cad5918c61a54bd50beb895c5b0fd60684 and it had the same issue. This is also true if I go back to v0.5.0. I also played around with various historical versions of ev3dev-lang-python and noticed that I only see the issue when on a version with our old folder structure. I am not exactly sure what the root cause is -- likely some combination of include/exclude settings and folder structure.

For reference, this is in my workspace settings:

```json
{
    "ev3devBrowser.download.include": "ev3dev/**",
    "ev3devBrowser.download.exclude": "{ev3dev/__pycache__/**,ev3dev/fonts/**}"
}
```

That may or may not be involved.
